ccpy/cc/ccsdt1_updates/t3a_100001.py

```diff
@@ -4,6 +4,7 @@
 
 import time
 
+#@profile
```

Comment:
Dangling comments like this should be turned into profiling functions that can be activated centrally from a "dev", "testing" or "profile" environment variable.